api/proto/common/v2/common_v2.proto

```diff
@@ -71,8 +71,14 @@ message PaymentHeader {
   // The account ID of the disperser client. This account ID is an eth wallet address of the user,
   // corresponding to the key used by the client to sign the BlobHeader.
   string account_id = 1;
-  // The reservation period of the dispersal request. 
+  // The reservation period of the dispersal request. The dispersal client's account will set this value to 
+  // current timestamp divided by the on-chain configured reservation period interval. The disperser server validates
+  // this value to be the period they receive this request, or from the previous period.
```

Comment:
also need to mention that one must have already paid for reserved bandwidth in order to make use of this, etc

Reply:
added some explanations